### PR TITLE
zh-translation: /docs/reference/glossary/service-mesh.md

### PR DESCRIPTION
**Pull request by wuti1609**
ref: https://github.com/servicemesher/istio-official-translation/issues/1037

### CODE REVIEW

**Comment by yuxiaobo96** [comment]
需根据提示链接签署CLA协议

**Comment by wuti1609** [comment]
@googlebot I signed it!

**Comment by wuti1609** [comment]
@googlebot I fixed it.

File: content/zh/docs/reference/glossary/service-mesh.md [hunk 1]
```diff
@@ -1,15 +1,8 @@
 ---
 title: Service Mesh
 ---
-A *service mesh* or simply *mesh* is an infrastructure layer that enables
-managed, observable and secure communication between
-[workload instances](/docs/reference/glossary/#workload-instance).
+一个 *服务网格* （简称 *网格* ）是一个可管理、可观测以及支持[工作负载实例](#workload-instance)之间进行安全通信的基础设施层。
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/service-mesh.md]: 去掉开头的”一个“

**Comment by wuti1609** [comment]
@googlebot I fixed it.

**Comment by wuti1609** [comment]
@googlebot I fixed it.

**Comment by wuti1609** [comment]
@googlebot I signed it!

**Comment by wuti1609** [comment]
@googlebot I fixed it.

**Comment by wuti1111** [comment]
@googlebot I consent.

File: content/zh/docs/reference/glossary/service-mesh.md (hunk 1 above)
Comment (wuti1111) [reply content/zh/docs/reference/glossary/service-mesh.md]: 已修改

**Comment by tanjunchen** [comment]
@malphi leaving a comment that contains only @googlebot I consent on #5823 Thanks

**Pull request merged**